src/screens/SendTokens.js

```diff
@@ -393,14 +471,37 @@ class SendTokens extends React.Component {
       }
     }
 
+    const renderSignTokensAlertBody = () => {
+      const tokenSignatures = tokens.getTokenSignatures();
+      const missingSigs = this.state.txTokens.filter(t => !(t.uid === '00' || tokenSignatures.hasOwnProperty(t.uid)));
+      const rows = [];
+      missingSigs.forEach(t => {
+        rows.push(<li><p>{t.name} ({t.symbol})</p></li>);
+      });
+
+      return <ul>{rows}</ul>
+    }
+
+    const renderLedgerModals = () => {
+      if (hathorLib.wallet.isSoftwareWallet()) return null;
+
+      return (
+        <div>
+          <ModalAlertNotSupported />
```

Comment:
I think we don't need this modal anymore, right?

Reply:
Removed